examples/common/Utilities.h

```diff
@@ -15,8 +15,7 @@ namespace Helpers
 {
     static inline void addAndMakeVisible (Component& parent, const Array<Component*>& children)
     {
-        for (auto c : children)
-            parent.addAndMakeVisible (c);
+        for (auto* c : children) parent.addAndMakeVisible (c);
```

Comment:
We don't write single-line for loops in this style (except where there are multiple loops on consecutive lines where lining them up vertically would be helpful to a reader).

Almost all the code in the repo is already formatted the way we like it, so if you see something like this, then maybe assume that we don't want it "fixed" unless you're 100% certain that it doesn't match the rest of the library.

Also, although in this case changing `auto` to `auto*` is fine, but there are many places in the code where `auto` is used in a for loop for possibly non-obvious but important reasons, so in general, steer clear of doing this unless it fixes a real problem!

Reply:
OK  I'll be more careful for the new line removal and respectful of whatever code style is in place.

Personally, I tend to avoid new lines without new brackets as I can't tell you how many bugs I found by people not careful of adding them. so I only allow myself to not add a bracket if the code is trivial and on the same line to avoid this:

if (my_enthusiastic_new_feature_works)
  TRACKTION_LOG("It should log this");
  TRACKTION_LOG("But not this unless I bug because it is indented but will always be called!");

Also for reading other's code, and for const and * alterations to auto: they help me a lot figuring what the code does and especially does not do (for const). This is part of the offical llvm recommendation BTW, see:
  https://releases.llvm.org/10.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/readability-qualified-auto.html

Reply:
Applied the suggested change, thanks for clarifying about teh coding style, we should probably add a .dox page on this ;-)